bin/configurer/utils/generate-local-user-mappings.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eu
+
+swanseauserids=$(docker run -e PGPASSWORD='openidm' --rm --network ccd-network postgres:11-alpine psql --host shared-db  --username openidm --tuples-only  --command "SELECT string_agg(fullobject->>'_id',',') FROM managedObjects WHERE fullobject->>'userName' LIKE '%swansea%';" openidm)
```

Comment:
To take it to next level you could use local function with one argument which is LA name. Function would return same outcome as it does now and function would be invoked 2 times, one time with `swansea` argument and second time with `hillingdon` argument.

It would allow us to cut on duplicates.

NICE TO HAVE.

Reply:
done